backend/routes/download.js

```diff
@@ -210,11 +209,7 @@ router.route('/zips/:showId')
             return submissionsWithSubmitters.reduce((arr, {user, group, entries}) => {
               // ['Clark Kent - Daily Planet Office']
               const newSubmissionSummaries = entries.map(({path, title, invited}) => {
-                const entryNamePrefix = (
-                  (user ? `${user.lastName} ${user.firstName}` : group.name) +
-                  ' - ' +
-                  title
-                )
+                const entryNamePrefix = `${(user ? `${user.lastName}, ${user.firstName}` : group.participants)} - ${title}`
```

Comment:
I think we definitely need to dedupe the group submissions. When something is a group submission we should just include that one (not a separate one for the creator too).

For group name I think we should do something like this (because I don't think the  group creator is listed in participants): 
`/Show Name/{Invited, Not Invited}/{creatorName}&Participants - Title <(num)>.jpg`

Reply:
@kayladavis Is the deduping problem solved? Do I just need to merge in master?

Reply:
I think for the purposes Nanette mentioned she uses this for, just the participants is fine. I'm okay will it being what you changed it to.

Reply:
Fixed it so that group creator is included